src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -273,8 +273,7 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
       (i) => {
         this.log.debug('Fetching L2BatchElement', { blockNo: startBlock + i })
         return this._getL2BatchElement(startBlock + i)
-      },
-      { concurrency: 50 }
+      }
```

Comment:
I'm a bit confused; so it used to have a concurrency of 50 and now this changes makes it have no limits at all?

Reply:
yup, the option only imposes a limit and the default is infinity: http://bluebirdjs.com/docs/api/promise.map.html#promise.map I missed the default last time so removing this now